src/app/app-routing.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Routes } from '@angular/router'
+import { authGuard } from "./core/auth/guards/auth.guard";
+import { loginPageGuard } from "./features/auth/guards/login-page.guard";
+
+export const routes: Routes = [
+  {
+    path: 'login',
```

Comment:
Not a good name for auth operations. Ok, Login page is under `/login`. But where the Sign In page will be? `/login/signin`?